api/environments/identities/models.py

```diff
@@ -19,6 +21,72 @@
         settings.IDENTITIES_TABLE_NAME_DYNAMO
     )
 
+dynamo_identifier_search_functions = {
+    "EQUAL": lambda identifier: Key("identifier").eq(identifier),
+    "BEGINS_WITH": lambda identifier: Key("identifier").begins_with(identifier),
+}
+
+
+class DynamoIdentityManager(models.Manager):
```

Comment:
This class should live in a `managers` module.

Reply:
In fact, does it even need to override the Manager class? What are we using from the `Manager` class?